R/mail.R

```diff
@@ -0,0 +1,65 @@
+# Kaiaulu - https://github.com/sailuh/kaiaulu
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+
+
+#' Creates fake mbox data
+#'
+#' Used to create a fake mbox with the specified parameters and returns the output to the console
+#'
+#' @param mlist specific mailing list associated with this mbox data
+#' @param reply_from_author name of sender. Formatted as: FIRSTNAME LASTNAME
+#' @param reply_from_email email of sender, automatically encapsulated in <> eg: sender@domain.com
+#' @param reply_to_author name of receiver. Formatted as: FIRSTNAME LASTNAME
+#' @param reply_to_email email of receiver, automatically encapsulated in <>  eg: receiver@domain.com
+#' @param reply_cc_author name of carbon copy of email to additional recipients. Formatted as: FIRSTNAME LASTNAME
+#' @param reply_cc_email email of carbon copy recipients, automatically encapsulated in <> input eg: recipientCC@domain.org
+#' @param reply_datetime Date, time in the following format: 2023-02-11T09:30:00  Be sure to add the capital T to separate date and time
+#' @param timezone  The abbreviation of desired timezone eg: HST
```

Comment:
My comment vanished when you made the last commits, but we still need to add a URL or what function to call here here telling the user where they can obtain information of the timezones they can type. Maybe the POSIX docs mentions it? You don't need to explain them here, but you should include a reference to where they can check.